tests/integrationv2/Makefile

```diff
@@ -28,7 +28,7 @@ endif
 define run_tox
 	( \
 	DYLD_LIBRARY_PATH="$(LIBCRYPTO_ROOT)/lib:$$DYLD_LIBRARY_PATH" \
-	LD_LIBRARY_PATH="$(LIBCRYPTO_ROOT)/lib:"$(S2N_ROOT)/test-deps/gnutls37/nettle/lib":$$LD_LIBRARY_PATH" \
+	LD_LIBRARY_PATH="$(LIBCRYPTO_ROOT)/lib:"$(S2N_ROOT)/test-deps/openssl-1.1.1/lib":"$(S2N_ROOT)/test-deps/gnutls37/nettle/lib":$$LD_LIBRARY_PATH" \
```

Comment:
why cant we just have `LD_LIBRARY_PATH="$(LIBCRYPTO_ROOT)/lib`? seems like we should always specify the `LIBCRYPTO_ROOT` in our testing. If that doesnt exist then failing seems like the better option rather than continuing with 111 or `gnutls37/nettle/lib` (which is default i assume?)

Reply:
We're using the `openssl` utility regardless of what `S2N_LIBCRYPTO` is set to, and ld can't find the correct libcrypto.so file.

Reply:
Alternative fix is to remove `OPENSSL_1_1_1_INSTALL_DIR` from the PATH [here](https://github.com/aws/s2n-tls/blob/main/codebuild/bin/s2n_setup_env.sh#L152), but this means we'll be using openssl3 on ubuntu22/AL2022 and I wasn't sure if that would cause other issues/concerns.